Gemfile

```diff
@@ -14,6 +14,7 @@ group(:development, :test) do
   gem "simplecov"
   gem 'rack', "~> 1.5.1"
   gem 'cheffish', "~> 1.3"
+  gem 'timecop'
```

Comment:
It could be moved to a dev dep, but I don't see the need for it since it brings in another layer of indirection and a _ton_ of monkey patches into the Time and DateTime classes that could affect other parts of the system inadvertantly.